command/agent/cache/lease_cache.go

```diff
@@ -814,3 +853,44 @@ func (c *LeaseCache) RegisterAutoAuthToken(token string) error {
 
 	return nil
 }
+
+type clearInput struct {
```

Comment:
The usage of `clearInput` and `parseClearInput` can be confusing. Can we use `CacheClearInput` and `parseCacheClearInput` respectively?

Reply:
I'll rename those but keep `cacheClearInput ` private since it's used for `handleCacheClear` which is also private.

Reply:
Yeah. Sounds good.